pxr/usdImaging/usdviewq/primViewItem.py

```diff
@@ -256,6 +260,10 @@ def _nameData(self, role):
                 toolTip = 'Inactive ' + toolTip
             elif self.isInstance:
                 toolTip = 'Instanced ' + toolTip
+        
+            # tooltip should always show both name and display name
+            toolTip = toolTip + "<br>Name: " + self.name + "<br>Display Name: " + self.displayName
```

Comment:
Thanks, @erslavin - I was not very clear in my request.  If a prim *has* a displayName, then we should show both name and displayName regardless of the "Show Display Names" setting.  But if a prim has **no** displayName, then I don't think we need to add anything here.

Reply:
Modified to conform to this behavior